specification/paths/Organizations.json

```diff
@@ -27,6 +27,14 @@
         "schema": {
           "type": "string"
         }
+      },
+      {
```

Comment:
Can we place this in its own file so we can reference it instead of declaring it twice? This will introduce some issues with the description I guess? What do you think.

Reply:
I was thinking the same thing, also with the issues with the description.

But I just thought that perhaps instead of saying that we search in the shop/organization's name, we can just say that we search in the resource's name attribute. wdyt?